src/ape/managers/config.py

```diff
@@ -58,6 +64,15 @@ def _plugin_configs(self) -> Dict[str, ConfigItem]:
         self.version = user_config.pop("version", "")
         self.dependencies = user_config.pop("dependencies", {})
 
+        # Sanitize deployment addresses.
+        deployments = user_config.pop("deployments", {})
+        for ecosystem_name, networks in deployments.items():
+            for network_name, contract_deployments in networks.items():
```

Comment:
What happens if `ecosystem_name` or `network_name` are not valid?

Reply:
Nothing was happening before, but now, it will raise a `ConfigError` if they are invalid.